pss/trojan/message.go

```diff
@@ -107,12 +107,12 @@ func (m *Message) Wrap(targets Targets) (chunk.Chunk, error) {
 	// 4064 bytes for trojan message + 32 bytes for nonce = 4096 bytes as payload for resulting chunk
 	binary.LittleEndian.PutUint64(span, chunk.DefaultSize)
 
-	chunk, err := m.toChunk(targets, span)
+	c, err := m.toChunk(targets, span)
 	if err != nil {
 		return nil, err
 	}
 
-	return chunk, nil
+	return c.WithType(chunk.ContentAddressed), nil
```

Comment:
type setting should be where validator is called no?

Reply:
a chunk is being created here, but sending it through a `localstore.Put` call should indeed invoke a `validate` func, and therefore set the chunk type.

on the receiving end, this should also be the case.

therefore i think this `WithType` call is probably unnecessary. 

i have removed it, but adjusted the tests so that the `WithType` is still invoked when we're not using the store validators.